net/librdkafka/Portfile

```diff
@@ -18,11 +18,42 @@ checksums           sha256 d6e553b1216b76de9e3a3c4b9c53fbecd87a5220aa5db2f3a8333
 
 patchfiles          CXX.patch
 
+configure.args-append --disable-zstd \
+                      --disable-lz4-ext \
+                      --disable-ssl \
+                      --disable-gssapi \
+                      --disable-syslog
+depends_lib           port:zlib
```

Comment:
Please use depends_lib-append

Reply:
Thank you, fixed by ec0076ba2092f774228589e17ff37896859bdca0